.github/workflows/rogue_ci.yml

```diff
@@ -166,12 +166,13 @@ jobs:
 
 # ----------------------------------------------------------------------------
 
-  docker_build_lib:
+  docker_build_rogue:
     needs: [full_build_test, small_build_test]
-    uses: slaclab/ruckus/.github/workflows/docker_build_lib.yml@main
+    uses: slaclab/ruckus/.github/workflows/docker_build_rogue.yml@main
```

Comment:
@slacrherbst wrong path: It should be `docker_build` (not docker_build_rogue)
https://github.com/slaclab/ruckus/blob/main/.github/workflows/docker_build.yml